src/app/body/create-new-sprint/create-new-sprint.component.html

```diff
@@ -74,7 +74,7 @@ <h4 class="modal-title" id="createNewSprintTitle">Create New Sprint</h4>
                         <div class="row pt-2">
                             <div class="col-md-4"></div>
                             <div class="col-md-4 pt-2 px-4">
-                                <button class="btn btn-secondary" type="button" data-dismiss="modal" (click)="close()">Close</button>
+                                <button class="btn btn-secondary" type="button" data-dismiss="modal" (click)="close();load()">Close</button>
```

Comment:
close();load()

What is the meaning of this.

Reply:
I created new function of name load() which reload whole page. when user click on start new sprint and created new sprint then this load() function is execute and if user only click on  start new sprint and click on cancel load() function is not execute